config/config.go

```diff
@@ -165,15 +177,25 @@ func GetDefaultConfigDir() (dir string, err error) {
 	return dir, nil
 }
 
-func expandPath(s string) string {
-	if len(s) >= 2 && s[0] == '~' && os.IsPathSeparator(s[1]) {
-		if runtime.GOOS == "windows" {
-			s = filepath.Join(os.Getenv("USERPROFILE"), s[2:])
-		} else {
-			s = filepath.Join(os.Getenv("HOME"), s[2:])
+// Given a path to either a file or directory, returns its absolute path format.
+// ExpandPath resolves "~/" prefix in a given system path.
+// Raise error if path is an empty string as it
+func ExpandPath(path string) (string, error) {
+	if path == "" {
+		error := errors.New("path to file/directory is not set.")
+		return path, error
+	}
+
+	if strings.HasPrefix(path, "~/") {
+		homedir, err := os.UserHomeDir()
+		if err != nil {
+			return path, err
 		}
+
+		return filepath.Join(homedir, path[2:]), nil
```

Comment:
I'd at least use a variable name for readability. Makes it even easier to see why path[:2]

`relativePath := path[:2]`

Reply:
That's reasonable